Jenkinsfile.d/core/weekly

```diff
@@ -18,7 +18,7 @@ pipeline {
   stages {
     stage("Release"){
       steps {
-        build job: 'core/release/master', parameters: [
+        build job: "core/release/${ BRANCH_NAME }", parameters: [
```

Comment:
do we need to do anything to stop the cron from being applied on a branch?

Reply:
Good catch, this is why I didn't remove the include('master') for the weekly release job as defined [here](https://github.com/jenkins-infra/charts/pull/265)

Reply:
but maybe we could add a safety net by adding a when condition right after the trigger instruction